R/api-config.R

```diff
@@ -0,0 +1,15 @@
+#' @title ApiConfig
+#' @docType class
+#' @description  Global configuration for the OSPSuite .NET API
+ApiConfig <- R6::R6Class(
+  "ApiConfig",
+  inherit = DotNetWrapper,
+  active = list(
+    pkParametersFilePath = function(value) {
+      private$wrapProperties("PKParametersFilePath", value)

```

Comment:
private is a keyword allowing me to create methods in base class that are not public available